docs/guides/typescript.md

```diff
@@ -55,7 +55,7 @@ const x = createFoo((_) => 'hello')
 
 Here again, `x` is `unknown` instead of `string`.
 
-One could argue that it is impossible to write an implementation for `createFoo`, and that would be true. But then it is also impossible to write Zustand's `create(...)`. However, Zustand exists.
+One could argue that it is impossible to write an implementation for `createFoo`, and that would be true. But then it is also impossible to write Zustand's `create` as a type. However, Zustand exists. So what do we mean by that?
```

Comment:
This still doesn't capture my intent, but for this PR it's okay because I have rewritten this section and I'll open a PR for it soon.